deployments/examples/ocis_full/.env

```diff
@@ -223,18 +223,16 @@ ONLYOFFICE_DOMAIN=
 WOPISERVER_ONLYOFFICE_DOMAIN=
 
 
-### Inbucket Settings ###
-# Inbucket is a mail catcher tool for testing purposes.
+### Mail Server Settings ###
+# Mailpit serves as both an email server and a mail catcher tool for testing purposes.
 # DO NOT use in Production.
 # Note: the leading colon is required to enable the service.
-#INBUCKET=:inbucket.yml
-# email server (in this case inbucket acts as mail catcher).
-# Domain for Inbucket. Defaults to "mail.owncloud.test".
-INBUCKET_DOMAIN=
-
+MAIL_SERVER=:mailserver.yml
```

Comment:
could we have some other name so that it easy to understand it's sonfig file, not a server